docs/source/overview/quicktour.md

```diff
@@ -183,7 +183,10 @@ Olive also can package output artifacts when user adds `PackagingConfig` to Engi
     ...
     "packaging_config": {
         "type": "Zipfile",
-        "name": "OutputModels"
+        "name": "OutputModels",
+        "extra_config": {
+            "export_model_in_mlflow_format": false
```

Comment:
No need to create "extra_config" sub dictionary. Just add a flag in the "packaging_config"